flyteadmin/pkg/manager/impl/metrics_manager.go

```diff
@@ -501,7 +501,7 @@ func (m *MetricsManager) parseNodeExecutions(ctx context.Context, nodeExecutions
 	return nil
 }
 
-// parseSubworkflowNodeExecutions partitions the SubworkflowNode execution into a collection of Categorical and
+// parseSubworkflowNodeExecution partitions the SubworkflowNode execution into a collection of Categorical and
```

Comment:
Thank you @sjtucoder 
Not sure I get the motivation for doing this, and I'm not clear on the effects